README.md

```diff
@@ -1,3 +1,5 @@
 # TextMining
 
 This is the base repo for the text mining and analysis project for Software Design at Olin College.
+
+The project writeup for my project can be found at: https://github.com/vickymmcd/TextMining/blob/master/TextMining%20(1).pdf.
```

Comment:
Writeup looks good, though you should remove the prompt text (like [Maximum 100 words]) as this might be seen by an external audience.